addons/sms/models/res_partner.py

```diff
@@ -7,3 +7,6 @@
 class ResPartner(models.Model):
     _name = 'res.partner'
     _inherit = ['mail.thread.phone', 'res.partner']
+
+    def _phone_get_number_fields(self):
+        return ['mobile']
```

Comment:
@datonefaridze 
We don't need to change this at all. It's already defined as ['mobile', 'phone'], that's the expected behavior

This breaks many tests

Reply:
's ok, we'll get there :D

To be fair the fact we use both phone and mobile by default without checking the format is a bit odd

Reply:
@reth-odoo done